depth/depth.go

```diff
@@ -244,12 +246,13 @@ func run(args dargs) {
 				cov = "NO_COVERAGE"
 			} else if depth < args.MinCov {
 				cov = "LOW_COVERAGE"
+			} else if args.MaxMeanDepth > 0 && depth >= args.MaxMeanDepth {
+				cov = "EXCESSIVE_COVERAGE"
 			}
 			// check for a gap or a change in the coverage class.
 			if cov != lastCov || pos != cache[1].pos+1 {
 				if lastChrom != "" {
-					stats := getStats(fa, lastChrom, cache[0].pos-1, cache[1].pos)
```

Comment:
Brent -- thanks so much for merging. Sorry about forgetting to mention the stats in the wrong file issue that was in this fix.